zoo-css-challenge/style.css

```
@@ -29,10 +31,31 @@ section {
   background-color: #f8f0fb;
 }
 
+#bear1 {
+  border: solid;
+  border-width: 5px;
+  border-color: red;
```

```suggestion
  border: 5px solid red;
```
Frontend developers would typically use the shorthand version for border like so